apps/rush-lib/src/cli/test/RushCommandLineParser.test.ts

```diff
@@ -0,0 +1,288 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+// Mock child_process so we can verify tasks are (or are not) invoked as we expect
+jest.mock('child_process');
+// Mock RushCommandLineParser itself to prevent `process.exit` to be called on failure
+jest.mock('../RushCommandLineParser', rushCommandLineParserMockFactory);
+
+import { resolve } from 'path';
+import { chdir } from 'process';
+import { ChildProcessModuleMock, ISpawnMockConfig } from 'child_process';
+import { FileSystem } from '@microsoft/node-core-library';
+import { Interleaver } from '@microsoft/stream-collator';
+import { RushCommandLineParser } from '../RushCommandLineParser';
+
+/**
+ * Interface definition for a test instance for the RushCommandLineParser.
+ */
+interface IParserTestInstance {
+  parser: RushCommandLineParser;
+  spawnMock: jest.Mock;
+}
+
+/**
+ * Helper to set up a test instance for RushCommandLineParser.
+ */
+function newCommandLineParserInstance(repoName: string, taskName: string): IParserTestInstance {
```

Comment:
Our convention is that function names should always start with an English verb

Reply:
Sounds good. Do you have a convention of which verbs to use when creating? `get` vs. `create` vs. something else?